samples/error-correction/bit-flip-code/BitFlipCode.qs

```diff
@@ -373,5 +374,5 @@ namespace Microsoft.Quantum.Samples.BitFlipCode {
         check(errors[1]);
```

Comment:
Would it make sense to use a loop here? Something like `for (error in [X0, X1, X2]) { check(error); }`? Assembling the constants into an array looks weird if that array is never used as an array.

Reply:
Thanks for taking a look, I've modified accordingly.